addons/purchase_stock/tests/test_create_picking.py

```diff
@@ -639,3 +639,35 @@ def test_update_qty_purchased(self):
         # check that the price has been updated in the purchase order line and in the stock.move
         self.assertEqual(purchase_order.order_line.price_unit, 0)
         self.assertEqual(purchase_order.picking_ids.move_ids.price_unit, 0)
+
+    def test_serial_tracking(self):
```

Comment:
This test should be in `stock`, not in `purchase_stock` (you don't need the `purchase.order` model , you can create a delivery (`stock.picking`) directly)

Reply:
@svs-odoo done 
thanks